envbuilder.go

```diff
@@ -525,10 +525,15 @@ func run(ctx context.Context, opts options.Options, execArgs *execArgsInfo) erro
 			if val, ok := os.LookupEnv("KANIKO_REGISTRY_MIRROR"); ok {
 				registryMirror = strings.Split(val, ";")
 			}
-			var destinations []string
+			destinations := []string{}
```

Comment:
I made this change because I was requested to do so by a linter, but perhaps I could have made it in a way that is not semantically different. Will check

Reply:
Which linter? `make lint` doesn't trigger this for me 🤔

Reply:
It was a linter in CI. I'll find the failure

Reply:
Ah! This makes sense now:
https://github.com/coder/envbuilder/actions/runs/11495941565/job/31996550118

I was forcing kaniko to output a tarball by adding a default item to this.
Without the default item, this can revert back to its original declaration.

Reply:
Fixed